gpii/node_modules/processReporter/README.md

```diff
@@ -0,0 +1,68 @@
+Process Reporter
+===
+
+A server app built deployed with a node.js server that returns process information.
+
+### Process Reporter API
+
+The app currently supports the following urls:
+
+    {urlToProcessReporterServer}/processes            // GET ONLY
+    {urlToProcessReporterServer}/processes/{solution} // GET ONLY
+
+The first form cycles through the solutions registry, and for each entry, determines if the associated processes are running.  The second form reports only the solution requested, for example:
+
+    {urlToProcessReporterServer}/processes/org.gnome.desktop.a11y.magnifier
+
+In both cases, the Process Reporter uses the information stored in the `isRunning` block of a solutions registry entry.  For example, the `isRunning` block for the Orca screen reader (the solution entry `org.gnome.orca`) is as
```

Comment:
Currently, the solution is ignored.  I don't think that's an error, but I'll ponder it some more.  I will look at various solutions and determine if there are ones where describing them as "running" is odd or nonsensical.  Perhaps there is a special case of "always running".

Also, I will add another special case  example to the README, which I noted in the JIRA.   Briefly, the linux magnifier is *not* a process.  The process it depends on is gnome-shell, but that isn't enough to tell if the magnifier is on.  In this case the g-s magnifier is running if (1) the gnome-shell process is running and (2) the gsetting is "on".  The linux implementation of the ProcessReporter takes that into account.